ideascube/serveradmin/wifi.py

```diff
@@ -171,7 +189,8 @@ def connected(self):
         for active_connection in NetworkManager.ActiveConnections:
             if (active_connection.Connection.GetSettings()
                     == self._connection.GetSettings()):
-                return True
+                if active_connection.State == NM_ACTIVE_CONNECTION_STATE_ACTIVATED:
```

Comment:
There is nothing preventing us from merging them.

I kept them separate as a matter of taste, since each one of them is already long enough to go past the 80-chars limit and needs to be continued on a new line.

I'm happy to merge them if you prefer, it makes no difference to me. :smiley: